httpMiddleware.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/palantir/go-githubapp/githubapp"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rs/zerolog"
+)
+
+type statusCodeResponseWriter struct {
+	http.ResponseWriter
+	statusCode int
+}
+
+func (w *statusCodeResponseWriter) WriteHeader(code int) {
+	w.statusCode = code
+	w.ResponseWriter.WriteHeader(code)
+}
+func InitializeResponder(ctx context.Context) context.Context {
+	var responder func(http.ResponseWriter, *http.Request)
+	return context.WithValue(ctx, responderKey{}, &responder)
+}
+
+type responderKey struct{}
```

Comment:
What is this used for?

Reply:
actually nothing. https://github.com/lunarway/release-manager-bot/pull/21/commits/5f3438aa5edf73b11c6414f2af55cf69242406d5